packages/falso/src/tests/aggregation.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { randAggregation } from '../lib/aggregation';
+
+describe('randPercentages', () => {
+  it.each([
```

Comment:
Please replace `it.each` with a for loop that checks this function 100 times.

Reply:
There is no meaning to this because you're not using the values in the test.

Reply:
🤦‍♂️ I missed that, thanks

Reply:
This is what I wanted to do:
```ts
...

(values, totalValue) => {
  const nums = randAggregation({
    totalValue: totalValue,
    values: values,
  });

...
```

Reply:
:)

And please run it 100 times per each to verify it working as expected